src/main/java/pl/lodz/p/it/zzpj/botsite/services/MessageService.java

```diff
@@ -3,18 +3,26 @@
 
 import pl.lodz.p.it.zzpj.botsite.entities.Message;
 import pl.lodz.p.it.zzpj.botsite.exceptions.entity.deletion.MessageDeletionException;
+import pl.lodz.p.it.zzpj.botsite.exceptions.entity.notfound.UserNotFoundException;
 import pl.lodz.p.it.zzpj.botsite.exceptions.entity.retrieval.MessageRetrievalException;
+import pl.lodz.p.it.zzpj.botsite.exceptions.entity.retrieval.UserRetrievalException;
 import pl.lodz.p.it.zzpj.botsite.exceptions.entity.saving.MessageAdditionException;
 import pl.lodz.p.it.zzpj.botsite.exceptions.entity.notfound.MessageNotFoundException;
+import pl.lodz.p.it.zzpj.botsite.exceptions.entity.saving.MessageUpdateException;
 
 import java.util.List;
 
 public interface MessageService {
 
-    Message addMessage(Message message) throws MessageAdditionException;
-    List<Message> getAllByUserId(String userId) throws MessageRetrievalException;
+    Message addMessage(Long userId, Message message) throws MessageAdditionException, UserNotFoundException;
```

Comment:
Zamiast tutaj userId dawać, to w Message dodaj referencję do User i tam go wpisz.